### PR TITLE
simplifying creation of ECR and S3 within jupyter notebook

### PR DESCRIPTION
**Pull request by dalbhanj**
*Issue #, if available:*https://github.com/aws-samples/eks-kubeflow-workshop/issues/12

*Description of changes:* added single command line and remove screenshot to simplify ECR repo creation from notebook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: notebooks/02_Fairing/02_01_fairing_introduction.ipynb [hunk 1]
```diff
@@ -176,6 +168,16 @@
     "!eval $(aws ecr get-login --no-include-email --region=$REGION)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an ECR repository in the same region\n",
```
Comment (Jeffwan) [inline notebooks/02_Fairing/02_01_fairing_introduction.ipynb]: What if people already have repository? it throws errors or something different? This repo is not just for workshop. Can we add one more step to check repo first and then ask you to create if it doesn't exist?

File: notebooks/02_Fairing/02_01_fairing_introduction.ipynb [hunk 2]
```diff
@@ -199,21 +201,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
```
Comment (Jeffwan) [inline notebooks/02_Fairing/02_01_fairing_introduction.ipynb]: Please create PY3 notebook

File: notebooks/02_Fairing/02_01_fairing_introduction.ipynb (hunk 1 above)
Comment (dalbhanj) [reply notebooks/02_Fairing/02_01_fairing_introduction.ipynb]: Added comment on this. Resolving

File: notebooks/02_Fairing/02_01_fairing_introduction.ipynb (hunk 2 above)
Comment (dalbhanj) [reply notebooks/02_Fairing/02_01_fairing_introduction.ipynb]: Done

**Comment by dalbhanj** [comment]
@Jeffwan I've added two more commits to improve 02_06_fairing_e2e.ipynb. Can you take a look?

**Pull request merged**